scania-digital-design-system/tegel
# feat(modal): Expose a closable property to be able to hide x icon

## **Describe pull-request**  
Expose a collapsible property to be able to hide x icon

## **Issue Linking:**  
- **Jira:** [CDEP-3294](https://tegel.atlassian.net/browse/CDEP-3294)    

## **How to test**  
1. Set closable property to false.
2. See that the close button is no more.

## **Checklist before submission**
- [] I have updated the documentation (if applicable)

## **Suggested test steps**

## **Additional context**  
Unit tests will be added if requested.


[CDEP-3294]: https://tegel.atlassian.net/browse/CDEP-3294?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
